project/currency_comparison/__tests__/index.test.js

```diff
@@ -0,0 +1,9 @@
+const index = require('../index.js')
+
+describe("Currency Comparison functionality", () => {
+    test("rejects non numeric salary entries", () => {
+        
+    })
+    test("rejects invalid currency codes", ()=>{})
+    test("returns response to proper input", ()=>{})
+})
```

Comment:
I don't think we need to test the index file. We can just focus on the currency_comparison / HTTP files.